### ISSUE TITLE
how does this work after the update?

### ISSUE THREAD

**Issue by Benoff** [tejado/pokemongo-api-demo#195]
I downloaded  the pgoapi master but what to do now?;(  please help

**Comment by elliottcarlson** [comment]
Review the [wiki documentation](https://github.com/tejado/pgoapi/wiki) and check the [pokecli.py](https://github.com/tejado/pgoapi/blob/master/pokecli.py) for an example of how to work with the API. Please ask for more specific help regarding issues.

**Comment by xnmrgn** [comment]
Yeah, I am completely confused as to how I can even run this program... I have python 2.7 installed but am not sure where to go from there.

**Comment by PieterxJan** [comment]
I came here after AHAAAAAAA shut down his repo. But it's not really clear to me how we can get this to work. I've also checked the wiki documentation, but it says that we need to run runserver.py. It used to be runserver.py with AHAAAAAAA's repo, but I can't find any runserver.py script in here?

**Comment by elliottcarlson** [comment]
This is an API, with an example script - the other project referenced utilized it, but other than that is unrelated. There are forks of that original repo that have continued - your best bet would be to find those.

**Comment by PieterxJan** [comment]
Ok, thanks for the clarification 👍
